ui/lib/config-ui/addon/routes/login-settings/index.js

```diff
@@ -4,5 +4,26 @@
  */
 
 import Route from '@ember/routing/route';
+import { service } from '@ember/service';
 
-export default class LoginSettingsRoute extends Route {}
+export default class LoginSettingsRoute extends Route {
+  @service api;
+  @service capabilities;
+  @service store;
+
+  async model() {
+    const adapter = this.store.adapterFor('application');
+
+    const { data } = await adapter.ajax('/v1/sys/config/ui/login/default-auth', 'GET', {
+      data: { list: true },
+    });
+
+    // this makes sense with data structure atm, but to be revisited
+    const loginRules = [];
+    data.keys.forEach((rule) => {
+      loginRules.push({ name: rule, namespace: data.key_info[rule].namespace });
+    });
```

Comment:
This logic is fairly common and @zofskeez made a util here: https://github.com/hashicorp/vault/blob/main/ui/app/services/api.ts#L187 if you want to try using that here